stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/StorageHelperWrapper.kt

```diff
@@ -58,7 +58,7 @@ public class StorageHelperWrapper(
                 upload = fileFromUri,
                 type = it.type,
                 name = it.title ?: fileFromUri.name ?: "",
-                fileSize = metaData.size,
+                fileSize = it.size.toInt(),
```

Comment:
I'm checking that this proper file size instead of attachment count doesn't break anything in the UI components...